### PR TITLE
Patch #81

### LINKED ISSUE #81: Install Dependabot and configure related settings
*This issue is part of a series of tasks listed in #81.*

### PR DESCRIPTION
**Pull request by Quantalabs**
# Patch #81
___

## New Feature
Add dependabot for dependency management. See #81 for more details.

## Known Issues
- #81 
- #79 
## Code Breakdown
Add dependabot, which checks for dependency updates, and creates a new branch and updates the dependency with a new commit prefixed `[dep]` or `[dep/dev]` for package dependencies and development dependencies, respectively. It then submits a PR, auto-assigning @quantum9Innovation, and adds the label [`dependencies`](https://github.com/epispot/epispot/labels/dependencies).
## Additional Notes
**Closes #81**

### CODE REVIEW

**Comment by quantum9Innovation** [comment]
@Quantalabs Fix the Dependabot file before review

**Comment by Quantalabs** [comment]
@quantum9Innovation waiting on review!

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      - "quantum9innovation"
+    commit-message:
+      prefix: "[dep]"
+      prefix-development: "[dep/dev]"
+      include: "scope"
+    labels:
+      - "dependencies"
+      - "low priority :arrow_down_small:"
+      - "unsupported :no_entry:"
+
```
Comment (quantum9Innovation) [inline .github/dependabot.yml:17]: Add pull request branch separator as `: `

**Review by quantum9Innovation — CHANGES_REQUESTED** [review]
Also add requirement install script to Travis CI

**Review by quantum9Innovation — APPROVED** [review]

**Pull request merged**